recipe/meta.yaml

```diff
@@ -48,9 +48,12 @@ about:
   license: MIT
   license_file: LICENSE
   summary: Store and access your passwords safely
+  dev_url: https://github.com/jaraco/keyring
+  doc_url: https://pypi.org/project/keyring/
 
 extra:
   recipe-maintainers:
     - licode
     - tacaswell
     - ccordoba12
+    - varlackc
```

Comment:
Any particular reason you want to be maintainer of this feedstock? Just asking because

* It's not really necessary to get your change accepted.
* Your change is quite simple and perhaps you wouldn't like to be linked to this feedstock in the future

Reply:
Those are very valid points, I have restored the maintainer's list to is original state